frontend/src/pages/notebookController/screens/server/SpawnerPage.tsx

```diff
@@ -417,6 +435,7 @@ const SpawnerPage: React.FC = () => {
             }
             setCreateInProgress(false);
           }}
+          isUsingCurrentTab={isUsingCurrentTab}
```

Comment:
We could pull this state from localStorage in the modal just to avoid passing it around. This state isn't exactly "form state" as it is application state.